src/Attestation/Attestation/ChangeLog.md

```diff
@@ -20,9 +20,11 @@
 
 
 ## Upcoming Release
+* Added three additional required assemblies to Az.Attestation.psd1
```

Comment:
This line is actually in S167 not S166, so you kept in under "upcoming release", right?

Reply:
yes